Sources/Native/NativeMockServerWrapper.swift

```diff
@@ -2,102 +2,206 @@ import Foundation
 import SwiftyJSON
 import BrightFutures
 
-class NativeMockServerWrapper: MockServer {
-  var port: Int32 = -1
-  var pactDir: String = ""
-
-  public init() {
-    pactDir = defaultPactDir()
-    port = randomPort()
-  }
-
-  func defaultPactDir() -> String {
-    if let dir = ProcessInfo.processInfo.environment["pact_dir"] {
-      return dir
+#if SWIFT_PACKAGE
+    import SwiftPactServer
+#endif
+
+public class NativeMockServerWrapper: MockServer {
+    private lazy var port: Int32 = {
+        return findUnusedPort()
```

Comment:
If this `findUnusedPort()` is only called here, why not have the logic encapsulated here? it's a private var too.

Reply:
TODO:
yes. I think this grew and probably was used somewhere else before.